### PR TITLE
organization activity feeds

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

File: ckanext/canada/controller.py [hunk 1]
```diff
@@ -458,11 +458,18 @@ def output_feed(self, results, feed_title, feed_description,
...
         )
 
+        chromos = h.recombinant_get_types()
+
         for pkg in results:
+            title, notes = None, None
+            if pkg['type'] in chromos:
+                chromo = h.recombinant_get_chromo(pkg['type'])
+                title= _(chromo['title'])
+                notes = _(pkg['notes'])
```
Comment (wardi) [inline ckanext/canada/controller.py]: these should have been copied onto the datasets, why is this necessary?

Comment (fanjinfei) [inline ckanext/canada/controller.py]: It is not there. See http://registry.open.canada.ca/en/api/2/rest/package/c3d6f6a9-9558-4376-9237-4573aee5f835

Comment (fanjinfei) [inline ckanext/canada/controller.py]: we translate it here https://github.com/open-data/ckanext-canada/blob/master/ckanext/canada/templates/public/snippets/package_item.html#L10

Comment (wardi) [inline ckanext/canada/controller.py]: they get updated by running `paster recombinant create <type>` after the description changes. Hacking it into the templates is something I'd like to avoid

Comment (wardi) [inline ckanext/canada/controller.py]: also it's better to ignore the rest API, it's going away in a future CKAN

Comment (fanjinfei) [inline ckanext/canada/controller.py]: maybe we should add `title_translated` here at https://github.com/open-data/ckanext-recombinant/blob/master/ckanext/recombinant/logic.py#L166 ?

Comment (wardi) [inline ckanext/canada/controller.py]: no we keep our title translations in po files so `paster recombinant create <type>` only needs to update the title field (as it does now)

Comment (fanjinfei) [inline ckanext/canada/controller.py]: With `paster recombinant create -f`, still the title has only English in solr. http://registry.open.canada.ca/api/action/package_show?id=c3d6f6a9-9558-4376-9237-4573aee5f835

Comment (wardi) [inline ckanext/canada/controller.py]: right and `h.get_translated` should use gettext to find the title for display because there is no title_translated field

Comment (fanjinfei) [inline ckanext/canada/controller.py]: So we will fix the h.get_translated() in ckan?

**Comment by wardi** [comment]
@fanjinfei is this complete?

**Comment by fanjinfei** [comment]
need the ckan PR: https://github.com/open-data/ckan/pull/99

**Comment by wardi** [comment]
I can't see a reason that it will be rejected. I'll merge that into ours
